src/kuzya.cpp

```diff
@@ -60,39 +66,7 @@ Kuzya::Kuzya(QWidget *parent)
     setObjectName("Kuzya");
     setWindowTitle("Kuzya");
     languageComboBox = new QComboBox(this);
-    toolBar = addToolBar("General");
-    toolBar->setObjectName("KuzyaToolBar");
-    toolBar->setIconSize(QSize(30,30));
-    toolBar->addAction(actionNew);
-    toolBar->addAction(actionOpen);
-    toolBar->addAction(actionSave);
-#ifdef Q_OS_MAC
-#else
-    toolBar->addSeparator();
-#endif
-    toolBar->addAction(actionUndo);
-    toolBar->addAction(actionRedo);
-    actionCut->setShortcuts(QKeySequence::Cut);
-    toolBar->addAction(actionCut);
-    actionCopy->setShortcuts(QKeySequence::Copy);
-    toolBar->addAction(actionCopy);
-    actionPaste->setShortcuts(QKeySequence::Paste);
```

Comment:
You need to configure key sequences for actions.

Reply:
Set it up in the designer